terraform/mgmtcluster.tf

```diff
@@ -99,6 +111,35 @@ write_files:
     owner: root:root
     path: /etc/needrestart/conf.d/needrestart.conf
     permissions: '0644'
+  - content: |
```

Comment:
Just as a side note. These lines would be more readable if each list entry would start with the path instead of the content.

Reply:
thanks, i included that now